jax/nn/__init__.pyi

```diff
@@ -0,0 +1,99 @@
+from typing import Any, List, Literal, Union, overload, Sequence
+
+import jax.numpy as jnp
+from jax._src.typing import (
+    Array, ArrayLike, DTypeLike
+)
+from jax._src.core import AxisName
+from jax._src.cudnn.scaled_matmul_stablehlo import BlockScaleConfig
+from jax._src.lax.lax import DotDimensionNumbers
+
+from jax.nn import initializers as initializers
+
+_Axis = Union[None, int, Sequence[int]]
+
+
+def identity(x: ArrayLike) -> Array: ...
```

Comment:
Could you please alphabetize all APIs in this file? It will make it easier to review and check for completeness.